service/orchestrator/orchestrator.go

```diff
@@ -53,7 +53,7 @@ var log *logrus.Entry
 
 var DefaultMetricsFile = "metrics.json"
 
-var DefaultCatalogsFile = "catalogs.json"
+var DefaultCatalogsFile = "demo_catalogs.json"
```

Comment:
we do not want to store the whole EUCS here ;)

Reply:
But we are not storing the whole EUCS catalog here. I only changed the naming from catalogs to demo_catalogs.json. The content is the same as before. Or do I miss something?

Reply:
You added the whole EUCS in service/orchestrator/eucs_catalogs.json

Reply:
Oh, you are right. I don't know what happened.... I will change it.